staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -4209,8 +4209,8 @@ func describeHorizontalPodAutoscalerV1(hpa *autoscalingv1.HorizontalPodAutoscale
 
 func describeNodeResource(nodeNonTerminatedPodsList *corev1.PodList, node *corev1.Node, w PrefixWriter) {
 	w.Write(LEVEL_0, "Non-terminated Pods:\t(%d in total)\n", len(nodeNonTerminatedPodsList.Items))
-	w.Write(LEVEL_1, "Namespace\tName\t\tCPU Requests\tCPU Limits\tMemory Requests\tMemory Limits\tAge\n")
-	w.Write(LEVEL_1, "---------\t----\t\t------------\t----------\t---------------\t-------------\t---\n")
+	w.Write(LEVEL_1, "Namespace\tName\t\tCPU Requests\tCPU Limits\tMemory Requests\tMemory Limits\tContainer Count\tAge\n")
```

Comment:
When we use kubectl describe node, we are more concerned about how many resources are on the node. It seems that the number of containers is not the main dimension we care about. To me, this change does not seem to be valid. If you care about the number of containers, you may be able to use a custom script to achieve it.